File: help/pt_pt/exercicios_da_ficha_11/Ficha11_009/Ficha11_009.c
// Programador           -

#include <stdio.h>
#include <string.h>
#include <math.h>
//User Defined Functions
void linha( int n );
void triangulo( int altura );

//Main Function
int main() {
    printf("Altura : ");
    int altura;
    scanf("%d", &altura);
    triangulo(altura ) ;
    return 0;
}

//::::::::::::::::::::::::::::::::::::::::::::::::::
//User Defined Functions Code
//::::::::::::::::::::::::::::::::::::::::::::::::::
void linha( int n ){
    int i = 0 ;
    for( int i = 0; i < n; i = i + 1 ) {
        printf("*");
    }
    printf("\n");
}
//::::::::::::::::::::::::::::::::::::::::::::::::::
void triangulo( int altura ){
    int i = 0 ;
    for( int i = 1; i < altura + 1; i = i + 1 ) {
        linha(i ) ;
    }
}


//C Language c11  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	quarta, 2022 out 12 20:10

File: help/pt_pt/exercicios_da_ficha_11/Ficha11_009/Ficha11_009.py
# Programador           -


#::::::::::::::::::::::::::::::::::::::::::::::::::
#User Defined Functions Code
#::::::::::::::::::::::::::::::::::::::::::::::::::
def linha(n):
    for i in range(0, n, 1 ):
        print( "*" , end="")
    print()

#::::::::::::::::::::::::::::::::::::::::::::::::::
def triangulo(altura):
    for i in range(1, altura + 1, 1 ):
        linha(i )

if __name__ == "__main__":
    altura =  int(input("Altura : "))
    triangulo(altura )


#Python 3  - Automatic translated by Algorithmi 22.05
#(c) Ant0nio M@nso 	quarta, 2022 out 12 20:10
